apps/engineering/content/design/components/id.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Id
+---
+
+import { ValueTruncateExample } from "./id.valueTruncate"
+import { WidthExample } from "./id.width"
+import { TypeTable } from 'fumadocs-ui/components/type-table';
+
+<TypeTable
+  type={{
+    value: {
+      description:
+        'The identifier to display.',
+      type: 'string',
+      default: undefined
```

Comment:
There is no default, it's a required prop
I guess this renders correctly but looks weird in code
```suggestion
```